tests/unit/modules/test_aptpkg.py

```diff
@@ -90,7 +90,26 @@
         'name': 'wget',
         'section': 'web',
         'source': 'wget',
-        'version': '1.15-1ubuntu1.14.04.2'
+        'version': '1.15-1ubuntu1.14.04.2',
+        'status': 'ii'
```

Comment:
Nitpick, but if you make this 
```suggestion
        'status': 'ii',
```
then if something else has to be added, it's only one line changed instead of two.